cli/README.md

```diff
@@ -34,3 +34,9 @@ help:
 ```bash
 popper --help
 ```
+
+To enable auto-completion support, you will need to activate the popper-complete.sh script
+inside the popper folder. This can be done by adding this into your bashrc :-  
+```bash
+ . /path/to/popper-complete.sh 
+``` 
```

Comment:
While this works, bash-completion is usually enabled by copying the script to `/etc/bash_completion.d/` or `/usr/local/etc/bash_completion.d/`. In our case, `/usr/local/bash_completion.d/` should work ok. Also the script has to be renamed `popper` when placed in these folders.

Reply:
Yes. I will make the changes. The same thing was specified on  [this answer](https://stackoverflow.com/a/18898614).